public/apps/configuration/panels/audit-logging/utils.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { displayArray, displayBoolean, displayMap } from '../../utils/display-utils';
+
+export function displaySettingType(setting, val) {
+  if (setting.type === 'bool') return displayBoolean(val);
+  else if (setting.type === 'array') {
+    return displayArray(val);
+  } else if (setting.type === 'map') {
+    return displayMap(val);
+  } else {
+    return 'Unknown type';
+  }
+}
+
+export function generateComboBoxLabels(arr) {
+  if (!arr) {
+    return;
+  }
+
+  return arr.map((value) => {
+    return { label: value };
+  });
+}
+
+export function removeComboBoxLabels(arr) {
+  if (!arr) {
+    return;
+  }
```

Comment:
There are already 2 functions [here](https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/dev/public/apps/configuration/utils/combo-box-utils.tsx#L22-L29). Besides the null check, I would do this [simple mapping](https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/dev/public/apps/configuration/panels/role-edit/tenant-panel.tsx#L52).

Reply:
Switched to the existing ultil functions.